webapp/components/test-file-results-table.js

```diff
@@ -276,6 +310,27 @@ class TestFileResultsTable extends WPTColors(PathInfo(TestRunsBase)) {
     }
     return `/api/screenshot/${shot}`;
   }
+
+  diffDisplay(results) {
+    if (results[0].status !== results[1].status) {
+      const passed = results.map(r => ['OK', 'PASS'].includes(r.status));
+      if (passed[0] && !passed[1]) {
+        return '↓ 1';
```

Comment:
IMHO, `↓` and `↑` are actually more confusing to me than simple `-` and `+`.

Reply:
```suggestion
        return '-1';
```